shell/store/index.js

```diff
@@ -464,12 +482,18 @@ export const mutations = {
     state.clusterReady = ready;
   },
 
-  updateNamespaces(state, { filters, all }) {
+  updateNamespaces(state, getters) {
+    const { filters, all } = getters;
+
     state.namespaceFilters = filters.filter(x => !!x);
 
     if ( all ) {
       state.allNamespaces = all;
     }
+
+    // Create map that can be used to efficiently check if a
+    // resource should be displayed
+    state.activeNamespaceCache = { 'test-performance': true }; // getActiveNamespaces(state, getters);
```

Comment:
I'm guessing this was meant to be switched back to `getActiveNamespaces` , once I did that everything seemed to work.

Reply:
Oops! I have changed it back.